service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/ManageDocumentsUploadedEventHandler.java

```diff
@@ -133,7 +133,8 @@ public void sendDocumentsByPost(final ManageDocumentsUploadedEvent event) {
                     .toList();
 
             if (!nonConfidentialPdfDocumentsToBeSent.isEmpty()) {
-                Set<Recipient> allRecipients = new LinkedHashSet<>(sendDocumentService.getStandardRecipients(caseData));
```

Comment:
Maybe add a unit test for this?

Reply:
will do! just popping it on here so I can test in preview